applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -187,51 +197,63 @@ int le_audio_send(uint8_t const *const data, size_t size)
 	static bool wrn_printed[CONFIG_BT_ISO_MAX_CHAN];
 	struct net_buf *buf;
 
+	/* TODO: This should all be based on number of audio channels */
+	uint8_t *p_data = (uint8_t *)data;
```

Comment:
I don't see the need for replicating this pointer here, see my other comment for how to use the data directly

Reply:
Done.